src/main/scala/lms/transformation/distributed_tensor/dim_name/TensorDimName.scala

```diff
@@ -64,6 +64,11 @@ abstract class DistributeTensorDimName extends Transformer with DataStructure {
     case Node(s, "tensor_weight", Backend.Const(tt:TensorType)::Backend.Const(anno:Anno)::_, _) =>
       implicit val sc_ : SourceContext = Adapter.oldSourceMap(s)
       WEIGHT(update_dim_name(tt), update_dim_name(anno)).x
+    
```

Comment:
It seems that the order of matching nodes is important. tensor_conv must come before other unary and binary op cases to avoid match exception
ok to put it here?